src/interpreter/kernel/exec.js

```diff
@@ -1,3 +1,5 @@
+import { debuglog } from "util";
```

Comment:
Ah yes good point. We could also use it here https://github.com/xtuc/js-webassembly-interpreter/blob/3dde1533bec38620ef95931dfa66955b9b798beb/src/interpreter/index.js#L199-L205 to trace the execution.